### ISSUE TITLE
Churn Images

### ISSUE THREAD

**Issue by PatrickJWalsh** [PatrickJWalsh/Mall-Customer-Segmentation#2]
RF Variable Importance - Balanced Data
![image](https://user-images.githubusercontent.com/71853253/95367460-cce1b400-08a2-11eb-8eab-1b48f7e90ef0.png)

Correlation Plot
![image](https://user-images.githubusercontent.com/71853253/95395132-e0eddb80-08cb-11eb-8537-a1c9f6ee824f.png)

International Plan Plot
![image](https://user-images.githubusercontent.com/71853253/95395842-8786ac00-08cd-11eb-898f-e124cdf2632c.png)

Voicemail Plan Plot
![image](https://user-images.githubusercontent.com/71853253/95396728-848cbb00-08cf-11eb-9658-48935e303c07.png)

Service Calls Histogram
![image](https://user-images.githubusercontent.com/71853253/95397904-68d6e400-08d2-11eb-8cc2-1156adcf1cda.png)

Churn by Voicemail
![image](https://user-images.githubusercontent.com/71853253/95398584-2ca48300-08d4-11eb-9ec0-b1ad8fe83850.png)

Churn by International Plan
![image](https://user-images.githubusercontent.com/71853253/95398988-22cf4f80-08d5-11eb-9bb1-4ffc6b7c1605.png)

Churn by Tenure
![image](https://user-images.githubusercontent.com/71853253/95400299-52338b80-08d8-11eb-9124-0b7d3782fad7.png)

Churn by Service Calls
![image](https://user-images.githubusercontent.com/71853253/95400457-a3dc1600-08d8-11eb-8452-24b264e5c611.png)

Log Regression Output
![image](https://user-images.githubusercontent.com/71853253/95401292-d850d180-08da-11eb-9aca-dab6a6ec6b19.png)

Deviance Table
![image](https://user-images.githubusercontent.com/71853253/95401659-cae81700-08db-11eb-8e3e-772576232ba6.png)

Confusion Matrix
![image](https://user-images.githubusercontent.com/71853253/95536972-618b0580-09ba-11eb-9093-293a666d1c7a.png)

Churn Prediction Probability
![image](https://user-images.githubusercontent.com/71853253/95403139-63cc6180-08df-11eb-9d44-2a75a01aee35.png)

Classification Accuracy by Threshold
![image](https://user-images.githubusercontent.com/71853253/95403373-143a6580-08e0-11eb-8e37-0afc6a273ea6.png)

ROC Curve
![image](https://user-images.githubusercontent.com/71853253/95404365-cecb6780-08e2-11eb-8d52-33fa2cea15af.png)

Tuned Predictions
![image](https://user-images.githubusercontent.com/71853253/95537026-8b442c80-09ba-11eb-8d50-f06d2891315b.png)

Cost by Business Case - Log Regression
![image](https://user-images.githubusercontent.com/71853253/95484010-eeec3c80-095d-11eb-85f5-23b377709970.png)

Forest Training Set Confusion Matrix
![image](https://user-images.githubusercontent.com/71853253/95492412-0aa91000-0969-11eb-8846-9ae3be32059b.png)

Forest test Set Confusion Matrix
![image](https://user-images.githubusercontent.com/71853253/95516095-b879e600-098c-11eb-8ae7-6878076d0be6.png)

Forest Number of Trees Tuning
![image](https://user-images.githubusercontent.com/71853253/95492854-af2b5200-0969-11eb-8589-6beb3f2ed6ef.png)

Forest Number of Variables to test Tuning
![image](https://user-images.githubusercontent.com/71853253/95493098-fd405580-0969-11eb-8ef5-3bb8f27bfcda.png)

Tuned RF Training set Confusion Matrix
![image](https://user-images.githubusercontent.com/71853253/95493271-41cbf100-096a-11eb-9f41-709cb0c66447.png)

Tuned RF Test Set Confusion Matrix
![image](https://user-images.githubusercontent.com/71853253/95516338-17d7f600-098d-11eb-9944-7dcbc20c6484.png)

Variable Importance Plot
![image](https://user-images.githubusercontent.com/71853253/95493826-05e55b80-096b-11eb-806d-4cf2210a5b89.png)

Variable Importance with Random Variable
![image](https://user-images.githubusercontent.com/71853253/95494784-750f7f80-096c-11eb-8946-6647e0190831.png)

Cost by Scenario with RF
![image](https://user-images.githubusercontent.com/71853253/95495620-ab99ca00-096d-11eb-8d2d-9829c1191fa2.png)

Balanced Data
![image](https://user-images.githubusercontent.com/71853253/95506884-92e5e000-097e-11eb-98be-9bce85b77850.png)

Logistic Regression on Balanced Data
![image](https://user-images.githubusercontent.com/71853253/95507174-0687ed00-097f-11eb-88e4-fe08cc4cc041.png)

Analysis of Deviance Table
![image](https://user-images.githubusercontent.com/71853253/95507528-8ca43380-097f-11eb-8772-0f4ecb6ed5e5.png)

Balanced Log Regression Prediction Confusion Matrix
![image](https://user-images.githubusercontent.com/71853253/95508453-07218300-0981-11eb-8688-2fa0da04dec8.png)

Balanced histogram of Predictions
![image](https://user-images.githubusercontent.com/71853253/95508977-d3932880-0981-11eb-97d6-453549c766b4.png)

Accuracy and Classification Threshold - Balanced Data
![image](https://user-images.githubusercontent.com/71853253/95511025-e22f0f00-0984-11eb-8549-6922afa944fb.png)

ROC Curve - Balanced Data
![image](https://user-images.githubusercontent.com/71853253/95511363-5cf82a00-0985-11eb-9530-faa5dba0dd0a.png)

Tuned Log Reg Predictions with Balanced Data
![image](https://user-images.githubusercontent.com/71853253/95511932-1d7e0d80-0986-11eb-9f65-0bf160e2327e.png)

Scenario Analysis with Balanced Data - no Forest
![image](https://user-images.githubusercontent.com/71853253/95512282-9d0bdc80-0986-11eb-852a-eeb90bdace39.png)

Training Set Confusion Matrix - Balanced Forest
![image](https://user-images.githubusercontent.com/71853253/95513938-1efd0500-0989-11eb-947f-7f7f97fd1291.png)

Test Set Confusion Matrix - Balanced Forest
![image](https://user-images.githubusercontent.com/71853253/95515820-38ec1700-098c-11eb-976a-de1876517097.png)

Number of Variable Test - Balanced Forest
![image](https://user-images.githubusercontent.com/71853253/95514461-f6293f80-0989-11eb-8931-a7a8ec719860.png)

Training Set - Tuned Forest on Balanced Data
![image](https://user-images.githubusercontent.com/71853253/95514647-47d1ca00-098a-11eb-8f13-b7f4ae2e2ccf.png)

Test Set Predictions - Tuned Forest on Balanced Data
![image](https://user-images.githubusercontent.com/71853253/95515661-ffb3a700-098b-11eb-83d5-52b92ee416c7.png)

Cost by Business Case - Balanced Data with Forest
![image](https://user-images.githubusercontent.com/71853253/95515078-0c83cb00-098b-11eb-819d-7b0b362d7e42.png)

Variable Importance - Balanced Forest
![image](https://user-images.githubusercontent.com/71853253/95517105-5e7a2000-098e-11eb-94a6-76c4246301ad.png)

% Churn based on # Service Calls
![image](https://user-images.githubusercontent.com/71853253/95536275-a57d0b00-09b8-11eb-8e28-9813465ffe3d.png)
